sqld/src/rpc/wal_log.rs

```diff
@@ -72,7 +72,8 @@ impl WalLog for WalLogService {
                 return Err(Status::failed_precondition(NO_HELLO_ERROR_MSG));
             }
         }
-        let start_offset = req.into_inner().start_offset;
+        // if current_offset is None, then start sending from 0, otherwise return next frame
+        let start_offset = req.into_inner().current_offset.map(|x| x + 1).unwrap_or(0);
```

Comment:
So this is a change in the behaviour. Was it a bug that we were sending frame from `start_offset` not `start_offset+1` before?

Reply:
no, before the client was asking for the frame that was after the last one it had received, now it tell the server what was the last frame it received and the server sends the next

Reply:
I see. This is really breaking new/old version compatibility.